src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -149,13 +150,15 @@ class JesBackend extends Backend with LazyLogging {
     val filesWithinExpressions = Try(
       call.task.outputs.map {
         taskOutput => taskOutput.expression.preevaluateExpressionForFilenames(scopedLookupFunction, engineFunctions)
-      }.flatMap({_.get})
+      }.flatMap({
```

Comment:
I think I have all of that stuff turned off or something.

OTOH my IntelliJ currently doesn't even remember that I have scala